Makefile.vars.mk

```diff
@@ -15,9 +15,12 @@ DOCKER_CMD ?= docker
 
 IMG_TAG ?= latest
 CONTAINER_REGISTRY ?= ghcr.io
+UPBOUND_CONTAINER_REGISTRY ?= xpkg.upbound.io
+
 # Image URL to use all building/pushing image targets
 CONTAINER_IMG ?= $(CONTAINER_REGISTRY)/$(PROJECT_OWNER)/$(PROJECT_NAME)/controller:$(IMG_TAG)
-PACKAGE_IMG ?= $(CONTAINER_REGISTRY)/$(PROJECT_OWNER)/$(PROJECT_NAME)/provider:$(IMG_TAG)
```

Comment:
Did you check the whole repository that there is no other occurrence of `PACKAGE_IMG`?

Reply:
yes

Reply:
I missed this one in the local-install target. I updated it from `PACKAGE_IMG` to `GHCR_PACKAGE_IMG`. Though I don't like the fact that it does not really push to GHCR during local install.

Reply:
yeah, for local-install this variable is set to the locally installed registry. If you prefix it to GHCR_ then the prefix doesn't make much sense for that local-install case :shrug:

Reply:
The easiest way is to remove GHCR from the name, do you have a better solution?

Reply:
not atm.